The path based redirect script has some errors

surajmb opened the pull request:
The redirect configuration should not be set in the Request Routing rule for a path based rule and must be set as a parameter in the Path rule config (New-AzureRmApplicationGatewayPathRuleConfig).
Also, we should not specify -BackendAddressPool and -BackendHttpSettings parameters in the path rule config when we configure redirection.

azurecla commented:
Hi __@stefsch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stefsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

surajmb committed:
The path based redirect script has some errors

The redirect configuration should not be set in the Request Routing rule for a path based rule and must be set as a parameter in the Path rule config (New-AzureRmApplicationGatewayPathRuleConfig).
Also, we should not specify -BackendAddressPool and -BackendHttpSettings parameters in the path rule config when we configure redirection.
